packages/newsroom-signup/src/Newsroom.tsx

```diff
@@ -153,18 +213,13 @@ class NewsroomComponent extends React.Component<NewsroomProps & DispatchProp<any
 
   constructor(props: NewsroomProps) {
     super(props);
-    let currentStep = props.address ? 1 : 0;
-    if (typeof props.initialStep !== "undefined") {
-      currentStep = props.initialStep;
+    let currentStep = props.address ? SECTION_STARTS[SECTION.CONTRACT] : 0;
```

Comment:
Oh yeah good call. I just replaced `1` with `SECTION_STARTS[SECTION.CONTRACT]` but ignored the rest of the line. Should be, um... I don't know if we have enough information to make a good guess unless we start looking inside the charter. I'll update to a better guess (`PROFILE_SO_FAR`), but since we require login before getting to this component at all, we can assume current step can be loaded from API, so when I add that in that'll override this.

Reply:
Actually, 0 is the safest guess - if this is their first time visiting there will be no saved step from API (and of course no newsroom address) so they should be at 0.